### ISSUE TITLE
Is the CrossEntropy of cleverhans different from caffe's softmaxwithloss?

### ISSUE THREAD

**Issue by SophieChang66** [tensorflow/cleverhans#1133]
I want to do adversarial training. So I train one two_class model on my own data using cleverhans, the final loss is about 0.2. However when I train the model with the same data, the final loss is about 0.01. The loss gap is large. I review the source code about CrossEntropy, I find it is tf.nn.softmax_cross_entropy_with_logits_v2. what difference between CrossEntropy and caffe's softmaxwithloss

**Comment by npapernot** [comment]
Unfortunately, we do not support caffe at this time so I am not sure what the difference would be between the TF implementation of the cross-entropy and caffe's implementation.

**Comment by SophieChang66** [comment]
> Unfortunately, we do not support caffe at this time so I am not sure what the difference would be between the TF implementation of the cross-entropy and caffe's implementation.
They are same! The loss is 0.2 is because smoothing_label parameter which change y label such(0, 1) to (0.95, 0.05). So the cross entropy is large
